src/config-hardware/audio-apps.md

```diff
@@ -0,0 +1,39 @@
+# Audio Applications & Extensions
+
+These applications provide control and customization settings for audio devices beyond the `Settings` menu.
+
+---
+
+## EasyEffects
+
+[EasyEffects](https://github.com/wwmm/easyeffects) is a third-party application that can apply audio effects and filters to audio input and output streams. You can download and install EasyEffects from the Pop!\_Shop.
+
+![EasyEffects](/images/audio-mic/easyeffects.png)
+
+## Audio Output Switcher
+
+[Audio Output Switcher](https://extensions.gnome.org/extension/751/audio-output-switcher/) is a GNOME extension that adds selectable audio input/output devices to the System menu. See [this section](/customize-pop/gnome-tweaks-extensions/gnome-extensions.md) for more information about installing GNOME Extensions.
+
+![Audio Output Switcher](/images/audio-mic/output-switcher.png)
+
+## Alsamixer
```

Comment:
I would capitalize this as `AlsaMixer` since that's what it shows at the top of the terminal in your screenshot.

Reply:
I've done this for all instances on this page.